tests/baselines/reference/dynamicRequire.js

```diff
@@ -4,6 +4,9 @@ function foo(name) {
 }
 
 //// [a_out.js]
-function foo(name) {
-    var s = require("t/" + name);
-}
+define("a", ["require", "exports"], function (require, exports) {

```

Comment:
hmm.. this does not look good. we should not mess up the module output if we can.

Reply:
not sure what to do though.. @rbuckton thoughts?

Reply:
This file has an explicit `@module: amd` BTW. Our module emit and our global emit for `commonjs` are pretty much identical - the only difference is in how the typechecker sees the top-level scope.